starboard/linux/x64x11/main.cc

```diff
@@ -35,26 +30,6 @@ extern "C" SB_EXPORT_PLATFORM int main(int argc, char** argv) {
   mallopt(M_ARENA_MAX, 2);
 
   tzset();
-  starboard::shared::signal::InstallCrashSignalHandlers();
-  starboard::shared::signal::InstallDebugSignalHandlers();
-  starboard::shared::signal::InstallSuspendSignalHandlers();
-
-#if SB_IS(EVERGREEN_COMPATIBLE)
```

Comment:
we should probably get rid of this flag at some point and use the USE_EVERGREEN one instead. Not in this change...